Makefile

```diff
@@ -52,6 +52,12 @@ pylint:
 	PYTHONPATH=.:./tcms/ DJANGO_SETTINGS_MODULE=$(DJANGO_SETTINGS_MODULE) pylint --load-plugins=pylint_django.checkers.migrations -d missing-docstring -d duplicate-code --module-naming-style=any  tcms/*/migrations/*
 	PYTHONPATH=.:./tcms/ DJANGO_SETTINGS_MODULE=$(DJANGO_SETTINGS_MODULE) pylint --load-plugins=pylint_django --load-plugins=kiwi_lint --load-plugins=pylint.extensions.docparams -d missing-docstring -d duplicate-code tcms/ tcms_settings_dir/
 
+.PHONY: pylint_site_packages
+pylint_site_packages:
+	if [ -d "$(PATH_TO_SITE_PACKAGES)" ]; then \
+		PYTHONPATH=.:./tcms/ DJANGO_SETTINGS_MODULE=$(DJANGO_SETTINGS_MODULE) pylint --load-plugins=kiwi_lint --disable=avoid-generic-foreign-key $(PATH_TO_SITE_PACKAGES);\
```

Comment:
this disables the new checker so it doesn't do anything useful.

You want all others disabled and only the new checker enabled here.

Reply:
I was only testing stuff out. Will switch it back